README.md

```diff
@@ -14,11 +14,22 @@ But the most important thing: **works on all OSs and all architectures ! Windows
 
 ## Install
 
-You can easily install IoTuring through pip. Although the version is os-specific, so follow the instructions below to install the right version for your operating system.
+## Who knows how it works
```

Comment:
This should be `###`, now it's the same level as `## Install`, looks strange

Reply:
Yeah, I also checked that, don't know how I haven't noticed it :(